pupil-spa/src/app/privacy/privacy.component.html

```diff
@@ -68,104 +68,106 @@ <h2 class="heading-small heading-contents" id="overview">Overview:</h2>
       <h2 class="heading-small heading-contents" id="legal">What is the legal basis for collecting and processing the data?</h2>
 
       <p>
-        The legal basis for processing the personal data used and collected during the 2018 MTC trial is covered by:
+        The legal basis for processing the personal data used and collected 
+        during the 2018 MTC trial is covered by:
       </p>
 
       <ul class="list list-bullet">
         <li>
-          Data Protection Act 1998, Schedule 2, section 5(c), which allows for when (data) processing is
-          necessary for the exercise of any functions of the Crown, a Minister of the Crown or a government department.
+          _Data Protection Act 2018, Part 2 Chapter 2 Lawfulness of processing Section 8 (d), 
+          which allows for when (data) processing is necessary for the exercise of a function of the Crown, 
+          a Minister of the Crown or a government department.        
         </li>
-
-        <li>
-          GDPR Article 6 (1)(e), which states that the processing is for the performance of a task carried
-          out in the public interest, or in the exercise of the data controller’s official authority, where
-          the processing is necessary for the exercise of a function of the Crown, a Minister of the Crown or
-          a government department and is a proportionate means of achieving a legitimate aim. This provision
-          covers personal data.
-        </li>
-      </ul>
-
-      <p>
-        Schedule 1 of the Data Protection Bill, which is currently making its way through Parliament, further defines how this condition can be met.
-      </p>
-
-      <ul class="list list-bullet">
         <li>
-          Schedule 1, part 2, 6 (1)(b) allows for when processing is necessary for reasons of substantial public interest.
-          Substantial public interest is defined by Schedule 1, part 2, 6 (2)(b) as being where the purpose is for the exercise
-          of a function of the Crown, a Minister of the Crown or a government department.
+          _GDPR Article 6 (1) (e), which states that the processing is for the performance of a task carried out in the public 
```

Comment:
Is the underscore deliberate? "_GDPR"

Reply:
@jon-shipley double checking this with David Kerr